app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java

```diff
@@ -0,0 +1,31 @@
+package io.github.droidkaigi.confsched2017.util;
+
+public class StringUtil {
+
+    public static boolean isHalfWidthCharacters(String s) {
```

Comment:
I think it's too much to check the characters 🤔 
If `lang` is null, it should not be bind `lang` to the view and it's better to hide it like this.
https://github.com/DroidKaigi/conference-app-2017/pull/115#discussion_r99573201

What do you think?